src/main/common/router/routablePath.ts

```diff
@@ -41,6 +47,10 @@ export class RoutablePath {
     const path = Object.entries(substitutions).reduce((uri: string, substitution: [string, string]) => {
       const [parameterName, parameterValue] = substitution
 
+      if (!RoutablePath.isValidParameterValue(parameterValue)) {
+        throw new Error(`Path parameter :${parameterName} is not provided`)
```

Comment:
It might be provided but invalid. Shall we say for example 'Path parameter :foo is invalid'

Reply:
done